packages/experimental/Checkbox/src/CheckboxTokens.ts

```diff
@@ -37,8 +35,11 @@ export const defaultCheckboxTokens: TokenSettings<CheckboxTokens, Theme> = (t: T
     checked: {
       checkmarkOpacity: 1,
     },
-    boxAtEnd: {
+    labelIsBefore: {
       checkboxMarginStart: 4,
       checkboxMarginEnd: 0,
     },
+    circular: {
+      borderRadius: 9999,
```

Comment:
We can't use `globalTokens.corner.radius.circle` here?

Reply:
Yeah I will convert this file to use globalTokens when I go through and make sure everything aligns to redlines.